zmon_worker_monitor/builtins/plugins/entities_wrapper.py

```diff
@@ -60,48 +56,61 @@ def __init__(self, service_url, infrastructure_account, verify=True, oauth2=Fals
             raise ConfigurationError('EntitiesWrapper improperly configured. URL is missing!')
 
         self.infrastructure_account = infrastructure_account
-        self.__service_url = service_url
+        self.__service_url = os.path.join(service_url, 'api/v1/')
```

Comment:
`os.path.join` is not really the right thing for URLs (it works, but it would not work on Windows... (not that we support win))